source/jsonnet/england-wales/ccs/blocks/who-lives-here/usual_address.jsonnet

```diff
@@ -0,0 +1,50 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+
+{
+  type: 'Question',
+  id: 'usual-address',
+  question: {
+    id: 'usual-address-question',
+    title: 'What was your household’s usual address?',
+    type: 'General',
+    answers: [
+      {
+        id: 'usual-address-answer-building',
+        label: 'Address line 1',
+        mandatory: false,
+        type: 'TextField',
+      },
+      {
+        id: 'usual-address-answer-street',
+        label: 'Address line 2',
+        mandatory: false,
+        type: 'TextField',
+      },
+      {
+        id: 'usual-address-answer-city',
+        label: 'Town or city',
+        mandatory: false,
+        type: 'TextField',
+      },
+      {
+        id: 'usual-address-answer-county',
+        label: 'County (optional)',
```

Comment:
It doesn't say (optional) on the google spreadsheet, is this supposed to be here?

Reply:
I'll remove it, I census schema's question was the example I used, hence 'optional'.